src/analysisd/alerts/exec.c

```diff
@@ -17,51 +17,29 @@
 #include "os_regex/os_regex.h"
 #include "os_execd/execd.h"
 #include "eventinfo.h"
+#include "wazuh_db/wdb.h"
 
+#define NEW_AR_MECHANISM "v4.2.0"
 
 void OS_Exec(int execq, int *arq, const Eventinfo *lf, const active_response *ar)
 {
-    char exec_msg[OS_SIZE_1024 + 1];
+    char exec_msg[OS_SIZE_8192 + 1];
+    char msg[OS_SIZE_8192 + 1];
     const char *ip;
     const char *user;
     char *filename = NULL;
     char *extra_args = NULL;
 
-    ip = user = "-";
-
     /* Clean the IP */
+    ip = "-";
     if (lf->srcip && (ar->ar_cmd->expect & SRCIP)) {
-        if (strncmp(lf->srcip, "::ffff:", 7) == 0) {
-            ip = lf->srcip + 7;
-        } else {
-            ip = lf->srcip;
-        }
-
-        /* Check if IP is to be ignored */
-        if (Config.white_list) {
-            if (OS_IPFoundList(ip, Config.white_list)) {
-                return;
-            }
-        }
-
-        /* Check if it is a hostname */
-        if (Config.hostname_white_list) {
-            size_t srcip_size;
-            OSMatch **wl;
-
-            srcip_size = strlen(ip);
-
-            wl = Config.hostname_white_list;
-            while (*wl) {
-                if (OSMatch_Execute(ip, srcip_size, *wl)) {
-                    return;
-                }
-                wl++;
-            }
-        }
+        ip = get_ip(lf);
+        if(ip == NULL)
```

Comment:
It is a good practice to add the `{}` anyway they are not needed:

```c
if (ip == NULL) {
    return;
}
```

Reply:
Please, apply this to all the cases below.

Reply:
Ok, changing. I usually don't use { } to clean up the code.